### PR TITLE
Update tutorial_controllers.rst

### PR DESCRIPTION
**Pull request by ejoful**
add function getSubscribedEvents()

### CODE REVIEW

File: development/extensions/tutorial_controllers.rst [hunk 1]
```diff
@@ -374,6 +374,14 @@ event listener should look like:
             $this->template = $template;
         }
 
+        static public function getSubscribedEvents()
+         {
+            return array(
+             'core.user_setup'	=> 'load_language_on_setup',
```
Comment (VSEphpbb) [inline development/extensions/tutorial_controllers.rst]: please fix the white spacing

Comment (ejoful) [reply development/extensions/tutorial_controllers.rst]: remove white spaces ，done

**Comment by VSEphpbb** [comment]
This PR should be made against the 3.1.x branch instead of 3.2.x, please.

**Comment by VSEphpbb** [comment]
Replaced with #157

**Pull request closed without merging**